packages/table/__tests__/table-core.test.ts

```
@@ -0,0 +1,346 @@
+            const lines = output.split("\n");
+
+            // Find the header line and data lines
+            const headerLine = lines.find((line) => line.includes("Tests"))!;
+            const unitLine = lines.find((line) => line.includes("Unit"))!;
+            const integrationLine = lines.find((line) => line.includes("Integration"))!;
+
+            // Extract cells from each line
+            const unitCells = unitLine.split("│").map((cell) => cell.replaceAll(/\u001B\[\d+m/g, ""));
+            const integrationCells = integrationLine.split("│").map((cell) => cell.replaceAll(/\u001B\[\d+m/g, ""));
+
+            // Verify Unit and Integration cells have same width
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)